pkg/plugins/handler_test.go

```diff
@@ -92,24 +92,24 @@ func TestLoadPTPPlugin(t *testing.T) {
 
 	testCases := map[string]struct {
 		pgPath  string
-		wantErr error
+		wantErr string
 	}{
 		"Invalid Plugin Path": {
 			pgPath:  "wrong",
-			wantErr: fmt.Errorf("ptp plugin not found in the path wrong"),
+			wantErr: "ptp plugin not found in the path wrong",
 		},
 		"Valid Plugin Path": {
 			pgPath:  "../../plugins",
-			wantErr: nil,
+			wantErr: "",
 		},
 	}
 
 	for name, tc := range testCases {
 		t.Run(name, func(t *testing.T) {
 			pLoader = Handler{Path: tc.pgPath}
 			err := pLoader.LoadPTPPlugin(wg, scConfig, nil)
-			if tc.wantErr != nil && err != nil {
-				assert.EqualError(t, tc.wantErr, err.Error())
+			if tc.wantErr != "" && err != nil {
+				assert.EqualError(t, err, tc.wantErr)
```

Comment:
Same here  This could have been single line change assert.EqualError(t,err ,tc.wantErr.Error())